modules/packages/TOML.chpl

```diff
@@ -370,6 +371,34 @@ module TomlParser {
           Date = d;
           return Date;
         }
+        // Time
+        else if ti.match(val) {
+          var raw = getToken(source).split(":");
```

Comment:
This works, but I'd like to suggest we use some built-in IO features to handle the rounding / edge cases, e.g.

```chpl
        else if ti.match(val) {
          var raw = getToken(source).split(":");
          var sec = '%.6dr'.format(raw[3]:real).split('.');

          var t: time;
          t = new time(raw[1]: int,
                       raw[2]: int,
                       sec[1]: int,
                       sec[2]: int);

          return t;
        }
```

Reply:
Note the formatted IO `%.6dr` should ensure there's always a second element in the `split('.')`

Reply:
well, didn't know that, this one's great actually.